transit_odp/timetables/loaders.py

```diff
@@ -281,17 +280,27 @@ def load_operating_profiles(self, vehicle_journeys):
             vehicle_journeys = vehicle_journeys.rename(
                 columns={"service_code_vj": "service_code"}
             )
-            merged_df = pd.merge(
-                vehicle_journeys[
-                    ["id", "vehicle_journey_code", "service_code", "file_id"]
-                ],
-                operating_profiles[
-                    ["vehicle_journey_code", "day_of_week", "service_code", "file_id"]
-                ],
-                on=["file_id", "service_code", "vehicle_journey_code"],
-                how="inner",
+            merged_df = (
+                pd.merge(
+                    vehicle_journeys[
+                        ["id", "vehicle_journey_code", "service_code", "file_id"]
+                    ],
+                    operating_profiles[
+                        [
+                            "vehicle_journey_code",
+                            "day_of_week",
+                            "service_code",
+                            "file_id",
+                        ]
+                    ],
+                    on=["file_id", "service_code", "vehicle_journey_code"],
+                    how="inner",
+                )
+                .dropna(subset=["day_of_week"])
```

Comment:
Could we remove this line as we are using query to filter out empty values

Reply:
Done